test/recipes/04-test_encoder_decoder.t

```diff
@@ -1,5 +1,5 @@
 #! /usr/bin/env perl
-# Copyright 2020 The OpenSSL Project Authors. All Rights Reserved.
+# Copyright 2021 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Wait, what?

Reply:
you could of done 2020-2021. Dont care either way.